File: c/PRAK605-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>

int main() {
    int ordo;
    scanf("%d", &ordo);

    int matriks_a[ordo][ordo];
    int matriks_b[ordo][ordo];

    printf("Matriks A\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            scanf("%d", &matriks_a[i][j]);
        }
    }

    printf("Matriks B\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            scanf("%d", &matriks_b[i][j]);
        }
    }

    printf("Matriks AXB\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            int total = 0;
            for (int k = 0; k < ordo; k++) {
                total += matriks_a[i][k] * matriks_b[k][j];
            }
            printf("%d ", total);
        }
        printf("\n");
    }

    return 0;
}

File: PRAK605-2510817210027-Ahmad_Tibrizi.py
ordo = int(input())
matriks_a = []
print("Matriks A")
for i in range(ordo):
    matriks_a.append(list(map(int, input().split())))

matriks_b = []
print("Matriks B")
for x in range(ordo):
    matriks_b.append(list(map(int, input().split())))

print("Matriks AXB")
for i in range(ordo):
    baris_hasil = []
    for j in range(ordo):
        total = 0
        for k in range(ordo):
            total += matriks_a[i][k] * matriks_b[k][j]
        baris_hasil.append(total)
    print(*baris_hasil)